docs/releases/3.2.4.txt

```diff
@@ -9,4 +9,5 @@ Django 3.2.4 fixes several bugs in 3.2.3.
 Bugfixes
 ========
 
-* ...
+* Prevented, following a regression in Django 3.2, unnecessary initialization
+  of unused caches (:ticket:`32747`).
```

Comment:
No need to break the flow of the sentence?
```suggestion
* Prevented unnecessary initialization of unused caches following a regression
  in Django 3.2 (:ticket:`32747`).
```

Reply:
Updated.